src/types/type-declarations.ts

```diff
@@ -17,7 +17,7 @@ export type HtmlAttributes = {
 
 type ScriptBase = {
   params?: Array<string>;
-  optionalParam?: Array<string>;
+  optionalParams?: Record<string, string | number | undefined>;
```

Comment:
should we include Null as well, since it's assigned as a default value in some scripts?

Reply:
oops i forgot this thanks !